src/components/screen/SearchUser.js

```diff
@@ -29,32 +39,179 @@ const styles: Styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+  viewSearch: {
+    width: '100%',
+    justifyContent: 'center',
+    // backgroundColor: colors.dodgerBlue,
+    overflow: 'hidden',
+  },
+  imgSearch: {
+    position: 'absolute',
+    width: 16,
+    height: 16,
+    left: 30,
+    top: 18,
+  },
 });
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
 type Props = {
-
+  navigation: any;
 };
 type State = {
-
+  users: Friend[]
 };
 
 class Screen extends Component<Props, State> {
   static navigationOptions = {
-    title: 'Title',
+    title: 'Search User',
   };
 
+  scrollY = new Animated.Value(0);
+  searchTxt: string = '';
+  users: Friend[];
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
```

Comment:
State also could uses class property syntax

```diff
-constructor(props) {
-  super(props);
-  this.state = { ... }
-}
+state = {
+  ...
+}
```

Reply:
When I remove `constructor` code as you adviced, **CircleCI** failed to test. so I added `constructor` again, **CircleCI** and **jest** test passed.
I'm newbie of **jest**, **enzyme**, so I will try to find out how to pass it without `constructor`.
Thank you for your review!!

Reply:
Yeah. Jest will fail without constructor.  It force to override the `props`.

Reply:
sounds like RN's jest preprocessor has unconfigured for "class properties" proposal :thinking: